src/components/auth/socialAuth.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import FacebookLogin from "react-facebook-login";
+import GoogleLogin from "react-google-login";
+import { connect } from "react-redux";
+import { facebookAuth, googleAuth } from "../../actions/authActions/socialAuth";
+import { Redirect } from "react-router-dom";
+import PropTypes from "prop-types";
+
+export class SocialAuth extends Component {
+  onFailure = error => {
+    alert(error);
+  };
+  responseGoogle = response => {
```

Comment:
Rename this function better to maybe handleGoogleResponse, or something along those lines

Reply:
Add tests for two these functions

Reply:
@LehruAngela  I have made this change